src/platform/unix/mod.rs

```diff
@@ -470,10 +470,21 @@ impl OsIpcReceiverSet {
 
     pub fn select(&mut self) -> Result<Vec<OsIpcSelectionResult>,UnixError> {
         let mut selection_results = Vec::new();
-        match self.poll.poll(&mut self.events, None) {
-            Ok(sz) if sz > 0 => {},
-            _ => { return Err(UnixError::last()); }
-        };
+        let mut sz = 0;
+        while sz <=0 {
```

Comment:
Nitpick: missing blank after `<=`.

BTW, since this is `usize`, I'd say it would be less misleading to just check `== 0`?